Gemfile

```diff
@@ -6,6 +6,9 @@ ruby "2.3.1"
 
 gem "rails", "~> 5.2.1"
 gem "bcrypt", "3.1.12"
+gem "faker", "1.7.3"
+gem "will_paginate", "3.1.6"
```

Comment:
Please, use kaminari gem. [https://github.com/kaminari/kaminari](https://github.com/kaminari/kaminari) for pagination